pkg/provisioner/ironic/ironic.go

```diff
@@ -931,6 +931,26 @@ func (p *ironicProvisioner) Deprovision(deleteIt bool) (result provisioner.Resul
 			nodes.ProvisionStateOpts{Target: nodes.TargetManage},
 		)
 
+	case nodes.Inpsecting:
+		p.log.Info("waiting for inspection to complete")
+		result.Dirty = true
+		result.RequeueAfter = introspectionRequeueDelay
+		return result, nil
+
+	case nodes.InspectWait:
+		p.log.Info("cancelling inspection")
+		return p.changeNodeProvisionState(
+			ironicNode,
+			nodes.ProvisionStateOpts{Target: nodes.TargetAbort},
+		)
+
+	case nodes.InspectFail:
+		p.log.Info("inspection cancelled")
```

Comment:
It can fail for other reasons as well, so this log may not always reflect what actually happened.

Reply:
I updated the message to say "failed or cancelled".